compiler/onecc-docker/onecc-docker

```diff
@@ -16,10 +16,43 @@
 
 import sys
 import subprocess
-import json
 import requests
 import os
 import argparse
+import re
+
+
+class RequestHandler:
+    def __init__(self, token=None, timeout=None):
+        if token:
+            self.headers = {"Authorization": "Bearer {}".format(token)}
+        else:
+            self.headers = {}
+        self.timeout = timeout or 5
+
+    def make_request(self, url):
+        try:
+            response = requests.get(url, headers=self.headers)
+            response.raise_for_status()
+            print("[onecc-docker] Request succeeded: {}".format(response.status_code))
+            return response
+        except requests.RequestException as e:
+            print("[onecc-docker] Request failed: ", e)
+            return None
```

Comment:
It is better to raise an exception for error cases so that the error case can be handled in main exception phrase.